configure-proxy-formula/configure-proxy/init.sls

```diff
@@ -0,0 +1,11 @@
+patterns-uyuni_proxy:
+  pkg.installed
+
+configure-proxy:
+  cmd.run:
+    - name: configure-proxy.sh --rhn-user={{ grains.get('server_username') | default('admin', true) }} --rhn-password={{ grains.get('server_password') | default('admin', true) }} --ssl-use-existing-certs
+    - requires:
+      - pkg: patterns-uyuni_proxy
+      - file: /root/config-answers.txt
```

Comment:
As Ondrej already said: where do you get the answer file from?
I would create a template and configure it via the pillar data

Reply:
Yes, I m now creating che answers.txt populate with pillar. Thanks for your help!